XMonad/Util/ExtensibleState.hs

```diff
@@ -90,6 +91,11 @@ modifyStateExts f = State.modify $ \st -> st { extensibleState = f (extensibleSt
 modify :: (ExtensionClass a, XLike m) => (a -> a) -> m ()
 modify f = put . f =<< get
 
+-- | Like @modify@ but the result value is applied strictly in respect to
+-- the monadic environment.
+modify' :: (ExtensionClass a, XLike m) => (a -> a) -> m ()
+modify' f = (\a -> let res = f a in res `seq` put res) =<< get
```

Comment:
This is pretty much the same as `(put $!) . f =<< get` right?

Reply:
Only if `a` is an instance of `NFData`, `$!` uses `deepseq` instead of `seq`.

Reply:
It is! I am changing it :+1: